src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithplaintextcommentfilter/InputSuppressWithPlainTextCommentFilterCustomMessageFormat.java

```diff
@@ -1,15 +1,41 @@
+/*
+SuppressWithPlainTextCommentFilter
+offCommentFormat = // CHECKSTYLE:OFF
+onCommentFormat = // CHECKSTYLE:ON
+checkFormat = (default).*
+messageFormat = .*tab.*
+idFormat = (default)(null)
+
+
+com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck
+id = ignore
+format = .*[a-zA-Z][0-9].*
+message = (default)
+ignoreCase = (default)false
+minimum = (default)0
+maximum = (default)0
+fileExtensions = (default)all files
+
+
+com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck
+eachLine = true
+fileExtensions = (default)
+
+
+*/
+
 package com.puppycrawl.tools.checkstyle.filters.suppresswithplaintextcommentfilter;
 
 public class InputSuppressWithPlainTextCommentFilterCustomMessageFormat {
 
     // CHECKSTYLE:OFF
-    private int A1;
-
-	private static final int a1 = 5; // contains tab character
+    private int A1; // violation
```

Comment:
Please put violation message to all lines if two Checks are in config
Or check has multiple messages

Reply:
done